### ISSUE TITLE
Quiet mobile 1.x users should be able to upgrade to 2.x with no problems

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#2041]
Mobile users of Quiet 1.x should be able to upgrade to 2.x. We're warning people that messages will be lost. 

* Ideally, data should not be lost; it should be recoverable. 
* Users should start Quiet 2.x with a clean install. 
* It is not required that users can access their Quiet 1.x data in Quiet 2.x.

**Comment by holmesworcester** [comment]
@EmiM @siepra any feedback on this?

**Comment by kingalg** [comment]
It looks like I have no way to check it. 

For android I can't change versions in the store so I can't install older app. 

For iOS I've installed old version (1.10.10 (317) - it was a version with popup about upcoming update), created community and send a message. Then I've installed 2.0.3 (338). I've expected thee app to be replaced by newer with clean data dir but Quiet updated to new version and loaded all messages. New messages can't be send - they are gray out.

![A0626E97-443C-43E1-9BFE-C787E2928189](https://github.com/TryQuiet/quiet/assets/68909168/f4798298-3021-46e5-9275-9f81f292bf2e)

@siepra any ideas what to do with this situation?

**Comment by siepra** [comment]
Im reopening #2094 and will check it on mobile

**Comment by kingalg** [comment]
@siepra what do you think? Let us know once you are back from holidays.

**Comment by holmesworcester** [comment]
To downgrade to the production version, the steps as I remember them are:

1. have two Google accounts on phone. One a tester, one not.
2. Switch to non tester account 
3. Uninstall quiet
4. Remove all Google play data (cache, stored data, etc)
5. Restart phone 
6. Check available version in Google play to confirm it is production

Then you can install it. It may be necessary to wait a day so it would be good to try this at the end of your day. There's some caching or delay so it might not be completely deterministic but it should work. If not, try different combinations of restarting and clearing the Google Play cache and try waiting longer.

@kingalg lmk if this works

**Comment by kingalg** [comment]
Upgrading on android is working - new app replace old one and is fully functional. 

I got an option to update from old to new version and it looks like it worked as replacing old app with new one as we wanted. 

I did it a few times and once one of a textures failed which resulted with this:
<img width="413" alt="Screenshot 2023-12-19 at 16 48 22" src="https://github.com/TryQuiet/quiet/assets/68909168/b8c0c5e2-18bf-4638-a759-5bb13c3b278a">

It happened directly after update so I'm reporting it here but I don't think that it's crucial and I'm not even sure if it's connected.
